E2E: remove cli fixtures in favor of e2e

tooppaaa opened the pull request.

tooppaaa committed:
e2e: add anuglar v7-lts

tooppaaa committed:
add versions to e2e

tooppaaa committed:
cleanup fixtures

tooppaaa commented:
@ndelangen if this is all green, should we proceed like this and forgot failing tests from the other PR ?

ndelangen committed:
DISABLE angular v6 e2e test

gaetanmaisse committed:
e2e: fix name of yarn2Cra tests config

gaetanmaisse commented on scripts/run-e2e-config.ts:
```diff
@@ -19,6 +19,14 @@ export const angular: Parameters = {
+export const angularv7: Parameters = {
+  ...baseAngular,
+  version: 'v7-lts',
+  additionalDeps: [...baseAngular.additionalDeps, 'core-js'],
```
I'm wondering why `core-js` is needed, can you give me some hints?

tooppaaa replied:
It's not "needed", but v7 (didn't checked above) installs core-js v2 which cause some issues with storybook

gaetanmaisse committed:
e2e: fix name of tests config to be valid npm project name

These names are used to init NPM project and so must follow validation rules:
https://github.com/npm/validate-npm-package-name

Moreover, they are now all consistant.

gaetanmaisse commented:
Just added 2 commits to fix Yarn 2 tests run, it was failing because we were trying to init project with a name invalid for NPM. Everything should be 🟢 .

Good job @tooppaaa 💪 👏
